index.bs

```diff
@@ -292,6 +292,42 @@ partial interface NeuralNetworkContext {
     Computes a 2-D convolution given input and filter 4-D tensors.
 </div>
 
+### gemm ### {#api-neuralnetworkcontext-gemm}
+<script type=idl>
+partial interface NeuralNetworkContext {
+  static Operand gemm(NeuralNetworkContext nn, Operand a, Operand b, Operand c, 
+                      optional float alpha = 1.0, optional float beta = 1.0, 
+                      optional boolean aTranspose = false, optional boolean bTranspose = false);
+};
+</script>
+<div algorithm=gemm>
+    **Arguments:**
+        - *nn*: The neural network context.
+        - *a*: an {{Operand}}. The first input 2-D tensor.
+        - *b*: an {{Operand}}. The second input 2-D tensor.
+        - *c*: an {{Operand}}. The third input 2-D tensor.
```

Comment:
May `c` be an optional argument?

Reply:
Yes. Please see the change in my new commit.

Reply:
It looks good. Thanks!